config/src/main/java/org/springframework/security/config/annotation/web/builders/WebSecurity.java

```diff
@@ -284,6 +292,7 @@ protected Filter performBuild() throws Exception {
 		for (SecurityBuilder<? extends SecurityFilterChain> securityFilterChainBuilder : this.securityFilterChainBuilders) {
 			securityFilterChains.add(securityFilterChainBuilder.build());
 		}
+		buildPrivilegeEvaluator(securityFilterChains);
```

Comment:
Why is this here the default instead of replacing the default for [`getPrivilegeEvaluator()`](https://github.com/spring-projects/spring-security/blob/a9d98e05f0471dbb888a48a60729279e053649d8/config/src/main/java/org/springframework/security/config/annotation/web/builders/WebSecurity.java#L252). Putting it in the getter ensures that if anyone tries to obtain it the defaulting is consistent.